composer/metrics/__init__.py

```diff
@@ -0,0 +1,9 @@
+"""A collection of common torchmetrics."""
```

Comment:
Similar question re: folder organization.

As an alternative, I'm wondering about a structure like this, where the loss functions, metrics, and model architectures are grouped together by model type. For example:

```
composer/models/
composer/models/nlp/huggingface_transformer_model.py
composer/models/nlp/metrics.py
composer/models/nlp/loss.py
```

I don't think I have a strong opinion either way, but am curious if others have thoughts or if it would be worthwhile to discuss during design review.

Reply:
talk about this with cory and matthew. We consider loss functions an area of research we are/should be exploring. metrics are separate from models. Am working with moin to make these even more general by moving some of the logic composermodels.